src/encoder.rs

```diff
@@ -347,6 +349,51 @@ impl<T: Pixel> FrameState<T> {
     }
   }
 
+  pub fn compute_activity_mask(&mut self) {
```

Comment:
If we could have documentation at this time itself, it will be very much helpful in future.
So it will be nice if we could add a few lines what this and subsequent function works in a nice way.